src/katgpucbf/fgpu/pfb.py

```diff
@@ -33,6 +33,13 @@
 class PFBFIRTemplate:
     """Template for the PFB-FIR operation.
 
+    The operation can operate in two different modes. In the first mode
```

Comment:
I feel funny (in a good way) about reading the words "This operation can operate".